ajax/libs/jquery-spellcheck/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "jquery-spellcheck",
+  "filename": "jquery.spellcheck.min.js",
+  "description": "A jQuery plugin that adds spellcheck support to inputs using the Google spell checker API.",
+  "keywords": [
+    "jQuery",
+    "plugin",
+    "spellcheck"
+  ],
+  "version": "0.2",
+  "author": {
+    "name": "Brandon Aaron",
+    "url": "http://brandon.aaron.sh/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/brandonaaron/jquery-spellcheck.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brandonaaron/jquery-spellcheck.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery-spellcheck*"
```

Comment:
@honestree Try to adjust the rule. You can try to clone [git auto-update](https://github.com/cdnjs/autoupdate) at the same path of cdnjs in your computer, then run `npm install` to install dependencies you need. Run `./autoupdate [library_name]` and you'll know files that auto-updater will fetch.